shishito/runtime/environment/shishito.py

```diff
@@ -138,36 +139,28 @@ def add_cmdline_arguments_to_browser(self, browser_capabilities, config_section)
             except:
                 pass
 
-    def add_download_path_to_browser(self, browser_capabilities, config_section):
-        """
-        add default path to downloading files from qa-bstack
-        :param browser_capabilities:
-        :param config_section:
-        :return:
-        """
+    def set_download_path(self, driver):
+        """set download path for Chrome. Variable in config file download_path can be either string or env var."""
         try:
-            browser_name = self.shishito_support.get_opt(config_section, 'browser').lower()
-            download_file_path = self.shishito_support.get_opt('download_path')
-            m = re.match('^\$([A-Z][A-Z_]+)$', download_file_path)
-
+            download_directory = self.shishito_support.get_opt('download_path')
+            m = re.match('^\$([A-Z][A-Z_]+)$', download_directory)
```

Comment:
If nothing is set in the `.properties` file, the `download_directory` is `None`, which will cause an Exception on this line  (`match()` does not accept `None`)

There should be a condition - if not setup in config file, ignore this completely